pkg/sriov/sriov.go

```diff
@@ -18,10 +19,17 @@ import (
 type NetlinkManager interface {
 	LinkByName(string) (netlink.Link, error)
 	LinkSetVfVlan(netlink.Link, int, int) error
+	LinkSetVfHardwareAddr(netlink.Link, int, net.HardwareAddr) error
+	LinkSetHardwareAddr(netlink.Link, net.HardwareAddr) error
 	LinkSetUp(netlink.Link) error
 	LinkSetDown(netlink.Link) error
 	LinkSetNsFd(netlink.Link, int) error
 	LinkSetName(netlink.Link, string) error
+	LinkSetVfTxRate(netlink.Link, int, int) error
```

Comment:
Vf support also min rate see [1]. It just not implemented in the netlink golang 

[1] - https://bugzilla.redhat.com/show_bug.cgi?id=1340914

Reply:
Yes, I had added that as a comment in types.go. There are currently 3 flags that are not supported in netlink lib, min_tx_rate, query_rss, state.